app/controllers/users_controller.rb

```diff
@@ -6,6 +6,7 @@ def new
   def create
     @user = User.new(user_params)
     if @user.save 
+      flash[:notice] = "User Created"
```

Comment:
Add the flash notice of "User Created" pag successful nakapag create ng user.
![image](https://user-images.githubusercontent.com/86232999/145022577-9fdd7a4b-caa6-4812-b49a-14dc33c474e2.png)